scripts/enforce-change-notes-updated-on-pr-to-master

```diff
@@ -15,12 +15,6 @@ if [[ ${CI_PULL_REQUEST-} && ${GITHUB_TOKEN-} ]]; then
 
   # PRs to master are typically either release or hotfix PRs, both of which should have change notes updated
   if [[ $base_branch == master ]]; then
-    if ! ./gradlew getChangelog -q --no-header &> /dev/null; then
-      die "Content of $change_notes_path has not been patched, fill the 'Unreleased' section and run ./gradlew patchChangelog"
-    fi
-    change_notes=$(./gradlew getChangelog -q)
-    if ! grep '^- ..*' <<< "$change_notes" &> /dev/null; then
-      die "$(head --lines=1 <<< "$change_notes" | sed 's/## //') section of $change_notes_path is empty or ill-formatted"
-    fi
+    ./gradlew verifyChangeLogContents > /dev/null
```

Comment:
```suggestion
    ./gradlew -q verifyChangeLogContents
```

Reply:
I wanted to keep the output of the script clean on success, but i guess displaying the progress bar for 3 seconds won't hurt.